ssz/src/main/java/tech/pegasys/artemis/ssz/backing/type/BasicViewTypes.java

```diff
@@ -11,31 +11,27 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package tech.pegasys.artemis.util.backing.type;
+package tech.pegasys.artemis.ssz.backing.type;
 
 import static com.google.common.base.Preconditions.checkArgument;
 
 import java.nio.ByteOrder;
 import org.apache.tuweni.bytes.Bytes;
 import org.apache.tuweni.bytes.Bytes32;
 import org.apache.tuweni.bytes.MutableBytes32;
-import tech.pegasys.artemis.util.SSZTypes.Bytes4;
-import tech.pegasys.artemis.util.backing.ViewRead;
-import tech.pegasys.artemis.util.backing.tree.TreeNode;
-import tech.pegasys.artemis.util.backing.view.BasicViews.BitView;
-import tech.pegasys.artemis.util.backing.view.BasicViews.ByteView;
-import tech.pegasys.artemis.util.backing.view.BasicViews.Bytes32View;
-import tech.pegasys.artemis.util.backing.view.BasicViews.Bytes4View;
-import tech.pegasys.artemis.util.backing.view.BasicViews.UInt64View;
+import tech.pegasys.artemis.ssz.backing.view.BasicViews;
+import tech.pegasys.artemis.ssz.SSZTypes.Bytes4;
+import tech.pegasys.artemis.ssz.backing.ViewRead;
+import tech.pegasys.artemis.ssz.backing.tree.TreeNode;
 
 /** The collection of commonly used basic types */
 public class BasicViewTypes {
 
-  public static final BasicViewType<BitView> BIT_TYPE =
+  public static final BasicViewType<BasicViews.BitView> BIT_TYPE =
       new BasicViewType<>(1) {
         @Override
-        public BitView createFromBackingNode(TreeNode node, int idx) {
-          return new BitView((node.hashTreeRoot().get(idx / 8) & (1 << (idx % 8))) != 0);
+        public BasicViews.BitView createFromBackingNode(TreeNode node, int idx) {
```

Comment:
Probably worth going back to importing `BitView` directly here instead of needing the prefix everywhere (suspect IntelliJ did this to you).

Reply:
Done.